### ISSUE TITLE
Retarget AppServices integration hosting startup to AspNetCore.App/All

### ISSUE THREAD

**Issue by pakrym** [aspnet/AzureIntegration#191]
And produce multiple versions of deps files for preview releases (rc1, rtm)

**Comment by muratg** [comment]
@pakrym Are we still doing something here in RC1?

**Comment by pakrym** [comment]
Nope, I have a prototype of what we might do if targeting AspNetCore.App is required but work would go into RTM after we decide what to fix in the host.

**Comment by DamianEdwards** [comment]
So, this doesn't need to be in rc1?

**Comment by muratg** [comment]
Approved for RTM.

**Comment by muratg** [comment]
@pakrym Any updates?

**Comment by pakrym** [comment]
Site extension is built, going to test right now.
